docs/uml/puml/ViewMangaSequence/refGetColumnsToPrintSequence.puml

```diff
@@ -0,0 +1,44 @@
+@startuml
+!include ../.PlantUMLSequence.config
+skinparam wrapWidth 0
+
+title <size:30>View Manga:Add Required Columns to List Sequence Diagram
+
+participant "MangaList" as ml <<class>>
+participant "row number" as row
+participant "manga name" as name
+participant "deadline" as dl
+participant "unit price" as price
+participant "units sold" as qty
+participant "total revenue" as rev
+note over row,rev : All classes below this note are of type <size:20>:PrintColumn<Manga>
+
+ml ++
+group sd [Add required columns to list]
+    ml -> row ** : add new PrintColumn
+    row ++
+    return added to columns
```

Comment:
I think you don't have to mention "added to columns" every time you return control, since you already mention "add new [column]" for each corresponding operation?

Reply:
You're right. Perhaps I can make it clearer by appending "to columns" to each top arrow.